tedana/decomposition/ica.py

```diff
@@ -50,6 +53,51 @@ def tedica(data, n_components, fixed_seed, maxit=500, maxrestart=10):
         "decompose the dimensionally reduced dataset."
     )
 
+    if ica_method=='robustica':
+        mmix, Iq = r_ica(data, n_components, n_robust_runs, maxit)
+        fixed_seed=-99999
```

Comment:
I presume the intention here is that if robust ICA is used, variable "`fixed_seed`" is meaningless. However the value "`-99999`" is not necessarily meaningless; it's an entirely meaningful integer value, and it's entirely possible that it could be used to seed an RNG depending on the API. It is more Pythonic to set the value of something that should not be used to `None`.

Reply:
This issue is fixed now as robustica is updated so that it achieves reproducible result with a seed.

Reply:
- Should post hyperlink to robustica issue describing resolution of the issue.
- I think here you're saying "the issue" in reference to the failure of robustica to honour a fixed seed. However the context of this specific conversation is inclusion of the "`fixed_seed=-99999`" line. So the response should better communicate what has changed in response to that comment.